nevergrad/optimization/test_optimizerlib.py

```diff
@@ -407,20 +407,20 @@ def test_parallel_es() -> None:
 @pytest.mark.parametrize(
     "dimension, num_workers, scale, budget, ellipsoid",
     [
-    (2, 3, 1., 120, False),
+    (2, 8, 1., 120, False),
     #(2, 1, 8., 120, True),
-    (2, 3, 8., 70, False),
+    #(2, 3, 8., 70, False),
```

Comment:
at some points the additional commented lines should just be removed

Reply:
I don't remove them as I do love them but I add an explanation.